_dashboards/branding.md

```diff
@@ -35,6 +35,8 @@ loadingLogo | Loading logo used when OpenSearch Dashboards is starting. See #3 i
 faviconUrl | Website icon. Loads next to the application title. See #4 in the image.
 applicationTitle | The application's title. See #5 in the image.
 
+**Note** --- To consolidate navigation controls and reduce the space the header takes up on the page, see [Condensed header](#condensed-header).
+
```

Comment:
For notes, you can use the Markdown {: .note } marker. Example:

This is a note.
{: .note }